app/jobs/import_contact_job.rb

```diff
@@ -0,0 +1,10 @@
+class ImportContactJob < ApplicationJob
```

Comment:
Would be useful to have a spec to check that the book goes thru the valid status updates.